devices/MulticastDNS/samples/Program.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Net;
+using System.Net.NetworkInformation;
+using System.Text;
+using System.Threading;
+using Iot.Device.MulticastDNS;
+using Iot.Device.MulticastDNS.Entities;
+using Iot.Device.MulticastDNS.Enum;
+using Iot.Device.MulticastDNS.EventArgs;
+using nanoFramework.Networking;
+using nanoFramework.WebServer;
+
+namespace MulticastDNS.Samples
+{
+    internal class Program
+    {
+        // Replace with your wifi ssid/pwd
+        const string Ssid = "...";
+        const string Pwd = "...";
+
+        // The following string contains the domain we will query through a browser.
+        const string DeviceDomain = "nanodevice.local";
+
+        private static string s_ipAddress;
+
+        public static void Main()
+        {
+            // Connect to the WiFi.
+            WifiNetworkHelper.ConnectDhcp(Ssid, Pwd);
```

Comment:
I know it's and example but maybe check the result and display a message that says there has been a problem if not connected correctly. It will definitely help beginners and people who will do mistakes in their wifi credentials

Reply:
Done